src/daft-local-execution/src/pipeline.rs

```diff
@@ -333,23 +333,44 @@ pub fn physical_plan_to_pipeline(
                         let right_size = right_stats.approx_stats.upper_bound_bytes;
                         left_size
                             .zip(right_size)
-                            .map_or(true, |(l, r)| (r as f64) < ((l as f64) * 1.5))
+                            .map_or(true, |(l, r)| ((r as f64) * 1.5) < (l as f64))
```

Comment:
I missed this final one: equality sign should flip, since if `1.5r < l` we want to build on the right and so `build_on_left` should be `false`.

Tbh the double negatives are kinda tripping me up (maybe there's a way to make this logic more natural).